src/client/graphics/layers/UILayer.ts

```
@@ -317,10 +317,16 @@ export class UILayer implements Layer {
         if (constructionType === undefined) {
           return 1;
         }
+        const constDuration = 
+          this.game.unitInfo(constructionType).constructionDuration;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)